code/places/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id("aep-library")
```

Comment:
Do we also need `id("com.android.library")` and `id("org.jetbrains.kotlin-android")`? I don't see this in the `aep-library`